### ISSUE TITLE
Idea: Alias option for locales in command

### ISSUE THREAD

**Issue by kitbs** [coderello/laravel-nova-lang#168]
Use case: A user wants to publish locale files differently from how they are named in the package.

Scenarios:
- Underscore instead of hyphen, e.g. `zh-CN` → `zh_CN`
- Favour script over region, e.g. `zh-CN` → `zh-Hans`
- Arbitrary, e.g. `en` → `english`?
- Clone an existing language to start editing a new variant, e.g. `de` → `de-CH`

Ideas:
- [x] New `--alias=` option for publish command with format such as `locale:alias(,locale:alias)*` 
- [x] Underscore or hyphen preference
- [x] Predefined mappings between common use cases:
  - `zh-CN`/`zh-TW` → `zh-Hans`/`zh-Hant` (favour script over region, as in CDLR)
  - `pt`/`pt-BR` → `pr-PT`/`pt` (favour Brazil as default over Portugal, as in CLDR)

**Comment by hivokas** [comment]
What about implementing "did you mean?" functionality. Seems to be relevant for this issue.

This function could be used: 
https://www.php.net/manual/ru/function.levenshtein.php

**Comment by kitbs** [comment]
I was just looking at levenshtein for something else today actually! 😄 I think that is a good idea to go along with #167 but I don't think it's quite right for this idea.

This use case here is that the user knows our file is named "zh-CN.json", but when they publish it to their own project, they want it to be saved as "zh-Hans.json". (Nowadays the script subtag is "more correct" than the region subtag for Chinese.)

So they could run the command something like:
```bash
php artisan nova-lang:publish zh-CN --alias=zh-Hans
```
or:
```bash
php artisan nova-lang:publish --all --alias=zh-CN:zh-Hans,zh-TW:zh-Hant
```

**Comment by hivokas** [comment]
Got it.

**Comment by kitbs** [comment]
Closed by #172
